modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+| "neo4j" | "standard" | []      | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | []           |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+=== Show the home database
+
+The home database for the current user can be seen using the command `SHOW HOME DATABASE`.
+
+.Query
+[source, cypher]
+----
+SHOW HOME DATABASE
+----
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name    | type       | aliases | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | constituents |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "neo4j" | "standard" | []      | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | []           |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+=== Filter the listed databases
+
+It is also possible to filter and sort the results by using `YIELD`, `ORDER BY`, and `WHERE`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name, currentStatus, requestedStatus
+ORDER BY currentStatus
+WHERE name CONTAINS 'o'
+----
+
+In this example:
+
+* The number of columns returned has been reduced with the `YIELD` clause.
+* The order of the returned columns has been changed.
+* The results have been filtered to only show database names containing `'o'`.
+* The results are ordered by the `currentStatus` column using `ORDER BY`.
+
+It is also possible to use `SKIP` and `LIMIT` to paginate the results.
+
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------+
+| name     | currentStatus | requestedStatus |
++--------------------------------------------+
+| "movies" | "online"      | "online"        |
+| "neo4j"  | "online"      | "online"        |
++--------------------------------------------+
+----
+
+[NOTE]
+====
+Note that for failed databases, the `currentStatus` and `requestedStatus` are different.
+This often implies an error, but **that is not always the case**.
+For example, a database may take a while to transition from `offline` to `online` due to a performing recovery.
+Or, during normal operation, a database's `currentStatus` may be transiently different from its `requestedStatus` due to a necessary automatic process, such as one Neo4j instance copying store files from another.
+The possible statuses are `initial`, `online`, `offline`, `store copying` and `unknown`.
+====
+
```

Comment:
I couldn't find any talk of/examples of listing composite database in the composite database section (there's probably some as part of like create/start/stop/drop but not on it's own). And the alias in composite listing examples are covered in other comments already.

In cypher manual we have this example (which for some reason is part of the example above and not it's own but I assume that is just a formatting error in the file):
![image](https://github.com/neo4j/docs-operations/assets/7591575/cf999826-4440-4e63-9b0e-1a092bb015f7)

I think having something like it would be nice

Reply:
Will fix when updating the composite databases page

Reply:
Have added.